### PR TITLE
snap: many changes

### PR DESCRIPTION
**Pull request by soumyaDghosh**
1. Populate with more metadata
2. Fix the desktop file name
3. Remove unused libraries

I am removing the libraries because they're statically linked and don't need externally. It can be checked by reading the elf files of the node modules

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -95,6 +63,7 @@ apps:
     extensions: [gnome]
     desktop: usr/share/applications/element-desktop.desktop
     command: launcher
+    autostart: element-desktop.desktop
```
Comment (kenvandine) [inline snap/snapcraft.yaml]: Can you please separate the autostart change into a separate PR?

**Review by kenvandine — CHANGES_REQUESTED** [review]
Please separate the autostart change into a separate PR.

File: snap/snapcraft.yaml (hunk 1 above)
Comment (soumyaDghosh) [reply snap/snapcraft.yaml]: Will it be okay, if I just edit the commit message?

Comment (kenvandine) [reply snap/snapcraft.yaml]: I'd prefer a separate PR. In case we want to revert that if we have negative feedback, it would be a simple revert.

Comment (soumyaDghosh) [reply snap/snapcraft.yaml]: Done, but I don't think that'd break anything.

Comment (kenvandine) [reply snap/snapcraft.yaml]: It won't break, but users might not want it to autostart by default. I want to consider that a bit before we merge that change.

**Review by kenvandine — APPROVED** [review]
Looks good now, thanks

File: snap/snapcraft.yaml (hunk 1 above)
Comment (soumyaDghosh) [reply snap/snapcraft.yaml]: No no, it supports the autostart feature. The app will not autostart by default. When users will enable the autostart on login option in the app, it'll place that desktop file under .config/autostart. If we recognize it as a valid autostrat file from here, snapd will allow the autostart.

https://snapcraft.io/docs/snapcraft-yaml-schema#p-21225-app-directives
